core/services/ocrcommon/adapters_test.go

```diff
@@ -2,15 +2,25 @@ package ocrcommon_test
 
 import (
 	"context"
+	"encoding/json"
 	"fmt"
 	"reflect"
 	"testing"
 
+	structpb "github.com/golang/protobuf/ptypes/struct"
```

Comment:
this package is deprecated. i think is part of the reason you were struggle to make a test.

see https://pkg.go.dev/github.com/golang/protobuf/ptypes/struct

` Deprecated: Use the "google.golang.org/protobuf" module instead.`

that pkg has a NewStruct func that is easier to use https://pkg.go.dev/google.golang.org/protobuf/types/known/structpb?utm_source=godoc#NewStruct